core/src/main/scala/akka/persistence/cassandra/query/QuerySettings.scala

```diff
@@ -41,4 +40,6 @@ import com.typesafe.config.Config
   val eventsByPersistenceIdEventTimeout: FiniteDuration =
     queryConfig.getDuration("events-by-persistence-id-gap-timeout", MILLISECONDS).millis
 
+  val healthCheckQuery: String = queryConfig.getString("health-check-query")
```

Comment:
if moving that to the `healthcheck` section it might make sense to include HealthCheckSettings in `PluginSettings`?

Reply:
Done